crates/crate_extractor/src/main.rs

```diff
@@ -77,6 +77,8 @@ fn main() {
             let text = serde_json::to_string(&json).unwrap_or_else(|err| {
                 panic!("Error while parsing ChangeJson object to string: {}", err)
             });
+            std::fs::create_dir_all(output.parent().unwrap())
```

Comment:
To be completely on the safe side, we'd have to make sure that `output` is a path to a file and not a directory. We could even rename the parameter to sth. like "outputFile" . And then check, if it's really a path to a file. What do you think?